tests/std/tests/P0896R4_ranges_alg_remove/test.cpp

```diff
@@ -21,26 +21,38 @@ struct instantiator {
 
     template <ranges::forward_range Read>
     static constexpr void call() {
-        using ranges::remove, ranges::subrange, ranges::iterator_t;
+        using ranges::remove, ranges::subrange, ranges::equal, ranges::iterator_t;
         { // Validate iterator + sentinel overload
             P input[5] = {{0, 99}, {1, 47}, {2, 99}, {3, 47}, {4, 99}};
             Read wrapped_input{input};
+            size_t comparisonsCounter = 0;
+            auto projection           = [&comparisonsCounter](const P& data) {
+                ++comparisonsCounter;
+                return data.second;
+            };

```

Comment:
Should we reuse this projection instead of duplicating to reduce the total number of required template instantiations? (Several occurrences.)

Reply:
The problem I had was that we need the `comparisonsCounter` reference. So I believe we can reuse it inside call but cannot put it in the instantiator itself

Reply:
I'll make this change. I'm digging into the test failures - it looks like something other than the `/analyze` and "references to pointers in constexpr" bugs is going on.